doc/ch-convert_desc.rst

```diff
@@ -31,7 +31,7 @@ producing the final format actually needed.
   :code:`-h`, :code:`--help`
     Print help and exit.
 
-  :code:`-i`, :code:`--in-fmt FMT`
+  :code:`-i FMT`, :code:`--in-fmt=FMT`, :code:`--in-fmt FMT`
```

Comment:
For documentation, I'd like to keep the short-hand that omits the argument for the short version; IMO that's sufficiently implied.

Similarly, I'd like to document either the equals or space, but not both, for conciseness. (My own quick impression is that I like the equals better, but I'm open to argument.) We could note somewhere that long arguments are documented one way but actually accept both.